rna/rna-alignment/README.md

```diff
@@ -0,0 +1,87 @@
+# RNAAlignment with STAR
+
+This workflow runs the STAR command on a pair of fastq files.
+(https://github.com/alexdobin/STAR) and produces an aligned, unsorted BAM
+with STAR's default compression level (1).
+
+This output BAM for this workflow can be sorted, compressed, and indexed
+with the `RNABAMSortAndIndex` workflow.
+
+Inputs:
+- Per-sample:
+  - Sample name
+  - R1 FASTQ file list
+  - R2 FASTQ file list
+
+- Reference:
+  - STAR index directory tar file (.tar.gz)
+
+- VM configuration
+  - docker image url
+  - VM disk size
+  - VM memory
+  - Runtime zones, ex: "us-central1-a us-central1-b"
+  - Number of times to try the workflow with a preemptible VM before
+    falling back to a full-price VM.
+
+Outputs:
+- Per-sample
+  - &lt;sample-id&gt;.Aligned.out.bam
+  - &lt;sample-id&gt;.SJ.out.tab
+  - &lt;sample-id&gt;.Chimeric.out.junction
+  - &lt;sample-id&gt;.Log.final.out
+  - &lt;sample-id&gt;.Log.out
+  - &lt;sample-id&gt;.Log.progress.out
+    - &lt;sample-id&gt;.STARgenome
```

Comment:
We never used the additional outputs from the `_STARgenome` and `STARpass1` directories.
I think that we should remove them from the published workflow.